src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -14,29 +14,83 @@
 
 package com.google.sps.servlets.utils;
 
-import com.google.sps.servlets.Range;
+import java.math.BigDecimal;
+import java.util.Comparator;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 public class Statistics {
-  // Aggregated score for a list of comments
+  private static final double INTERVAL = 0.2;
+  private static final double UPPER_END = 1.0;
+  private static final double LOWER_END = -1.0;
+  // Contains sentiment scores in the range [-1, 1] with given intervals.
   private Map<Range, Integer> aggregateValues;
   private double averageScore;
 
+  public Map<Range, Integer> getAggregateValues() {
+    return aggregateValues;
+  }
+
+  public double getAverageScore() {
+    return averageScore;
+  }
+
   public Statistics(List<Double> sentimentScores) {
+    sentimentScores =
+        sentimentScores.stream()
+            .filter(score -> (score >= LOWER_END && score <= UPPER_END))
+            .collect(Collectors.toList());
```

Comment:
nit: Align the "." in this line and the next line with the "." from the line above.

Reply:
Hi Cis, quick question here.... the built-in linter keeps moving my set alignment back to a different version.... what should we do here...

Reply:
![image](https://user-images.githubusercontent.com/42793737/86177620-97336f00-baec-11ea-8701-b115c2491b8d.png)

Reply:
Let linter do its thing then :) You can't fight it :P